### ISSUE TITLE
MultiTaskLassoCV got an unexpected keyword argument 'normalize'

### ISSUE THREAD

**Issue by anhtvan** [microsoft/SparseSC#47]
Hi, I'm getting an issue when trying to run this code. As far as I know the data is correclty formatted. The type error relates to some LassCV backend - could someone help me please? 
![image (3)](https://user-images.githubusercontent.com/91666191/218586016-7901bdc5-780d-4d0f-afbd-83a2efc30127.png)

**Comment by sak-18** [comment]
Same issue.

**Comment by sak-18** [comment]
https://scikit-learn.org/1.1/modules/generated/sklearn.linear_model.MultiTaskLassoCV.html?highlight=multitasklassocv Looking at this doc seems like normalize argument was deprecated and later removed altogether from scikit-learn 1.2. Downgrading sci-kit-learn should do, I hope for now.

**Comment by tom-flamelit** [comment]
@sak-18 which version of scikit-learn do you recommend?

**Comment by VanAltrades** [comment]
@samuelgarca 
I just ran into this unexpected 'normalize' argument as well.

I cannot confirm the root cause, but I was able to solve this error by modifying the `_MTLassoCV_MatchSpace` function.
I accessed the function via this path:

![path to matchspace](https://github.com/microsoft/SparseSC/assets/133554543/ceaf6088-b10a-4c69-89a9-6ca1ecfc08e1)

All I did was comment out the lines containing `normalize=normalize` and replace them with the same line sans `normalize=normalize`.

![comment out normalize](https://github.com/microsoft/SparseSC/assets/133554543/12d814f5-898d-4875-bf31-75290ad24ab2)

Since this modification, I have been able to run `SparseSC.fit_fast` without errors.

I'm not knowledgeable enough about versioning and compatibility to explain the root cause, but hopefully, this tip will help you get the package operational.

Jack

**Comment by MJeremy2017** [comment]
I think the sklearn dependency is obsolete. In the newer version this parameter is already removed

**Comment by franc703** [comment]
I have done the same as @VanAltrades and run the code without any problems. Make sure you normalize the data before.
